test/services/pull/index.test.ts

```diff
@@ -11,6 +11,7 @@ import { SigInfoSchema } from "../../../src/config/SigInfoSchema";
 import sigInfoSchema from "../../../src/config/sig.info.schema.json";
 import { PullFormatQuery } from "../../../src/queries/PullFormatQuery";
 import { Status } from "../../../src/services/reply";
+import {migrateToJSONTip} from "../../../lib/services/messages/PullMessage";
```

Comment:
Please combine this import with line 4-8 and it should be from src.

Reply:
Good catch! Thanks for your review.